utilities/storage.py

```diff
@@ -99,6 +99,17 @@ def create_dummy_first_consumer_pod(volume_mode=DataVolume.VolumeMode.FILE, dv=N
         )
 
 
+def adjust_size_based_on_sc(storage_class, requested_size_gi=5):
+    storage_class_dict = get_storage_class_dict_from_matrix(storage_class=storage_class)
+    storage_class = [*storage_class_dict][0]
+    volume_mode = storage_class_dict[storage_class]["volume_mode"].lower()
+    if volume_mode == "filesystem":
+        overhead_percent = 20
+        return requested_size_gi * (1 + overhead_percent / 100)
+
+    return requested_size_gi
```

Comment:
I'm not sure why we need it. 
If we want to fix `test_public_registry_data_volume_low_capacity`, please pass the size explicitly in the test. 
But it all looks like a product bug.